sql/memo/memo.go

```diff
@@ -305,6 +307,20 @@ func (m *Memo) memoizeBindvar(e *expression.BindVar) *ExprGroup {
 	return grp
 }
 
+func (m *Memo) memoizeBetween(e *expression.Between) *ExprGroup {
```

Comment:
i think we might normalize `Between` into the two halves, might have to check for the Lt/Gt combos

Reply:
I don't think we do, because in my tests, we are indeed matching on Between expressions and generating the SlidingRangeJoin plan.

However, we probably want to check for the Lt/Gt combos anyway, because Between is always inclusive on both ends, and the client query isn't. This was just faster to initially only support Between.

Reply:
We now support parsing both Between and LT/GT pairs, see the new PR for details.